.pre-commit-config.yaml

```diff
@@ -38,6 +40,7 @@ repos:
     hooks:
       - id: mypy
         # To avoid dependencies redefinition.
+        exclude: tests/
```

Comment:
it's better to put it into `pyproject.toml` into `[tool.mypy]` section  

note: you'd probably need to change this hook to smth like this

```yaml
- id: mypy
  language: system
  entry: mypy gendiff --config-file pyproject.toml
```